src/forms/spaceEditSchema.ts

```diff
@@ -48,5 +48,6 @@ export const spaceEditSchema = Yup.object().shape({
         : schema.notRequired()
   ),
   maxBooths: Yup.number().required(),
-  boothTemplateSpaceId: Yup.string().notRequired(),
+  // .nullable() is added for backwards compatibility after 'null' value was wrongly stored in some documents
```

Comment:
[Question] Are there a lot of these? Maybe we can modify this value manually/via migration?

Reply:
> Are there a lot of these?


I don't know. Firebase can't filter a field by `null` value. We can run a script in all of the environments to change the `null` to `""` which would work fine IMO.

Reply:
Firebase seems to be very unforgiving when it comes to `null` values and we should avoid them as much as possible.

Reply:
There were only 5 documents in staging and sparkle 7. I changed them manually and this fix should be enough.